extensions/logging-json/runtime/src/main/java/io/quarkus/logging/json/runtime/JsonConfig.java

```diff
@@ -51,4 +54,22 @@
      */
     @ConfigItem
     boolean printDetails;
+    /**
+     * Override keys with custom values. The special string "default" indicates that no key overrides will be applied.
+     */
+    @ConfigItem(defaultValue = "default")
+    String keyOverrides;
+
+    /**
+     * Keys to be excluded from the Json output. The special string "none" indicates that no keys will be excluded.
+     */
+    @ConfigItem(defaultValue = "none")
+    Set<String> excludedKeys;
```

Comment:
Wouldn't it make more sense for the empty set to mean "none"?

Reply:
Mapped to `Optional<Set<String>>` that should have the desired semantics.

Reply:
Not sure why, but discovered that a `Map` in config can be `null` but a `null` `Set` result in:

```
 SRCFG00040: The config property quarkus.log.console.json.excluded-keys is defined as the empty String ("") which the following Converter considered to be null: io.smallrye.config.Converters$CollectionConverter
 ```